.circleci/config.yml

```diff
@@ -1036,27 +1036,27 @@ jobs:
           name: "yarn-project"
           command: yarn-project/deploy_npm.sh latest
 
-  deploy-dockerhub-canary:
-    machine:
-      image: ubuntu-2204:2023.07.2
-    resource_class: medium
-    steps:
-      - *checkout
-      - *setup_env
-      - run:
-          name: "Deploy to dockerhub"
-          command: yarn-project/deploy_dockerhub.sh canary
+  # deploy-dockerhub-canary:
+  #   machine:
+  #     image: ubuntu-2204:2023.07.2
+  #   resource_class: medium
+  #   steps:
+  #     - *checkout
+  #     - *setup_env
+  #     - run:
+  #         name: "Deploy to dockerhub"
+  #         command: yarn-project/deploy_dockerhub.sh canary
 
-  deploy-ecr:
-    machine:
-      image: ubuntu-2204:2023.07.2
-    resource_class: medium
-    steps:
-      - *checkout
-      - *setup_env
-      - run:
-          name: "yarn-project"
-          command: yarn-project/deploy_ecr.sh
+  # deploy-ecr:
```

Comment:
We need this step for our current devnet deployments. These will shortly be made from CCI.

Reply:
I'm hoping for us to remove all "deployments to ecr".
Now we're fully open source we can just deploy to dockerhub.